portable-atomic-util/src/arc.rs

```diff
@@ -2292,7 +2302,8 @@ impl<T, const N: usize> From<[T; N]> for Arc<[T]> {
     /// ```
     #[inline]
     fn from(v: [T; N]) -> Self {
-        // Casting Arc<[T; N]> -> Arc<[T]> requires unstable CoerceUnsized, so we convert via Box.
+        // Casting Arc<[T; N]> -> Arc<[T]> requires unstable CoerceUnsized (with Rust nightly),
+        // so we convert via Box to support stable & beta Rust channels.
```

Comment:
removing "(with Rust nightly)":

```suggestion
        // Casting Arc<[T; N]> -> Arc<[T]> requires unstable CoerceUnsized,
        // so we convert via Box to support stable & beta Rust channels.
```

This may look like smaller line width in comparison to other places, which I think is too minor to worry about here.

Reply:
We do not depend on CoerceUnsized by default even on nightly, so I think the original text should be correct.

Reply:
now reverted